pkg/config/app_config.go

```diff
@@ -338,6 +338,7 @@ keybinding:
     diffingMenu: 'W'
     diffingMenu-alt: '<c-e>'
     copyToClipboard: '<c-o>'
+    copyCommitMessageToClipboard: '<c-y>'
```

Comment:
If you rebase on master you'll need to add this to KeybindingUniversalConfig in `pkg/config/user_config.go` like so:
```
CopyCommitMessageToClipboard              string `yaml:"copyCommitMessageToClipboard"`
```

Reply:
we'll also want to copy and paste this line into `docs/Config.md`